topics/go/algorithms/data/bloom/bloom.go

```diff
@@ -0,0 +1,78 @@
+package bloom
+
+import (
+	"hash/maphash"
+	"math/big"
+	"math/rand"
+)
+
+// Bloom is an implementation of a bloom filter.
+type Bloom struct {
+	m    int      // Number of bits
```

Comment:
is m and k idiomatic for this?

Why not called these what they are and then we don't need a comment?